### ISSUE TITLE
chown error with read-only mount volume

### ISSUE THREAD

**Issue by cdalexndr** [blacktop/docker-logstash-alpine#9]
Using docker compose servvice with `/usr/share/logstash/pipeline/` read-only mount point to provide configuration files inside container results in:
```
chown: /usr/share/logstash/pipeline/logstash.conf: Read-only file system
chown: /usr/share/logstash/pipeline: Read-only file system
```

The original docker image doesn't have this issue.

**Comment by blacktop** [comment]
I'm confused because you marked the filesystem READ ONLY and then are trying to write the file's ownership?  What is the original? the official one?

**Comment by cdalexndr** [comment]
I'm using unaltered images to prevent unnecessary image repository storage, and I'm mounting the configuration files using docker-compose:
```
volumes:
      - ./config/logstash/pipeline/:/usr/share/logstash/pipeline/:ro
      - logstash-data:/usr/share/logstash/data
```
I'm guessing the error comes from `7.3/logstash-entrypoint.sh` when calling `chown -R logstash: /usr/share/logstash`.

The official `logstash:7.3.0` image works with read-only mounted configuration folder, thus I cannot use your alpine image as an easy plug-in replacement.

**Comment by mk2020-star** [comment]
Hi,
  I am also facing the same problem, Please let me know if there is any solution.  

Thanks,

**Comment by cdalexndr** [comment]
As a workaround I've reverted to using the official image: https://hub.docker.com/_/logstash

**Comment by mk2020-star** [comment]
Thanks for the response. But official image is centos based and i need to use alpine based image.

**Comment by blacktop** [comment]
@mk2020-star my last commit pulled this image closer inline with the way the official images work, so your issue might be resolved in the 7.5/7.5.2 version of this image.
